src/main/resources/templates/kotlin-lang/kotlinClassGraphqlType.ftl

```diff
@@ -11,6 +11,11 @@ import ${import}.*
 <#if toStringForRequest>
 import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLRequestSerializer
 </#if>
+<#if generateModelOpenClasses>
```

Comment:
@jxnu-liguobin please merge nested if statement:
```ftl
<#if generateModelOpenClasses && equalsAndHashCode>
```

(relevant to all .ftl classes)